code/web/src/modules/user/EditProfile.js

```diff
@@ -0,0 +1,67 @@
+import React, { Component } from 'react'
+import { connect } from 'react-redux'
+
+
+
+class EditProfile extends Component {
+    constructor(props) {
+        super()
+        this.state = {
+            //image: ?
+            name: '',
+            description: '',
+            email: '',
+            address: ''
+        }
+    }
+
+    handleUserInput = (event) => {
+        this.setState( { [event.target.name]: event.target.value } )
+    }
+
+    render() {
+        return(
+            <form>
+                <input
+                    type='text'
+                    name='name'
+                    placeholder='name'
+                    value={this.state.name}
+                    onChange={event => this.handleUserInput(event)}
+                />
+                <input
+                    type='text'
+                    name='description'
+                    placeholder='description'
+                    value={this.state.description}
+                    onChange={event => this.handleUserInput(event)}
+                />
+                <input
+                    type='text'
+                    name='email'
+                    placeholder='email'
+                    value={this.state.email}
+                    onChange={event => this.handleUserInput(event)}
+                />
+                <input
+                    type='text'
+                    name='address'
+                    placeholder='address'
+                    value={this.state.address}
+                    onChange={event => this.handleUserInput(event)}
+                />
+                <button type='button'>Submit</button>
+                {/* onClick, will need to dispatch action*/}
```

Comment:
I'm thinking this should a `PATCH` request so that only on function has to run to update the `user` info.